changelogs/fragments/fortios-migration-removal.yml

```diff
@@ -0,0 +1,26 @@
+removed_features:
+- |
+  The ``fortimanager`` httpapi plugin has been removed from this collection.
+  It was a duplicate of the one in the `fortinet.fortimanager <https://galaxy.ansible.com/fortinet/fortimanager>`_ collection.
+  If you use ansible-base 2.10 or newer, a redirection has been provided.
+
+  If you use Ansible 2.9 and installed this collection, you need to adjust the FQCNs (``community.network.fortimanager`` → ``fortinet.fortimanager.fortimanager``) and make sure to install the `fortinet.fortimanager` collection.
+- |
+  All FortiOS modules and plugins have been removed from this collection.
+  They have been migrated to the `community.fortios <https://galaxy.ansible.com/community/fortios>`_ collections, respectively.
```

Comment:
```suggestion
  They have been migrated to the `community.fortios <https://galaxy.ansible.com/community/fortios>`_ collection.
```

Reply:
Ha, thanks for spotting that! In the beginning I started editing the fragment to mention both moves, but then I split it up into two parts. Apparently I forgot to undo that edit :)